### ISSUE TITLE
Error: mount_macfuse: the macFUSE kernel extension is not loaded

### ISSUE THREAD

**Issue by AlexC127** [osxfuse/osxfuse#851]
after upgrade to MacOS 12 on MBA M1,  I have upgraded macFUSE to 4.2.1
Security has been granted.
macFUSE is able to use only once using Veracrypt.
In the next attempt, it prompts the subject error. 
I have to reboot the MBA  everytime

**Comment by ericswpark** [comment]
@AlexC127, this might be a duplicate of #850. Can you check if the unload kext command [in this comment](https://github.com/osxfuse/osxfuse/issues/850#issuecomment-967795637) works for you or not?

**Comment by DaveFlowberries** [comment]
I have the same issue when trying to mount an existing volume or create a new volume:

"mount_macfuse: the macFUSE kernel extension is not loaded"

Tried with Veracrypt 1.25.7 and 1.24 versions, same issue. 
System is:
M1 Max MacBook Pro
OSX 12.1 (21C52) 

Only workaround I found is to run this every time I need to mount or create a volume:

sudo kextunload -b io.macfuse.filesystems.macfuse

That will work for one session, after which I need to run the command again.

**Comment by DaveFlowberries** [comment]
Updating OSX to 12.2 (21D49) solved the issue for me.

EDIT: Never mind, still have the issue after a day of using it without issues....

**Comment by GuillaumeHullin** [comment]
For me the following steps work: 
1. Verify that Terminal has Full Disk Access: <kbd>System Preference</kbd> > <kbd>Security & Privacy</kbd> > <kbd>Privacy</kbd> > <kbd>Full Disk Access</kbd>
1. Run `sudo kextunload -b io.macfuse.filesystems.macfuse`

macOS 12.1

**Comment by oldbinky** [comment]
Thanks [GuillaumeHullin](https://github.com/GuillaumeHullin)
That worked

**Comment by GuillaumeHullin** [comment]
You're welcome @oldbinky 
@AlexC127 I guess you can close this issue?

**Comment by SophieOstmeier** [comment]
Had the same problem.
Resolved after restarting my Mac.
Good luck!

**Comment by AlexC127** [comment]
Thanks @GuillaumeHullin,  It works.   
I can now skip the "sudo kextunload -b io.macfuse.filesystems.macfuse" command now.

**Comment by SpeedyLSN** [comment]
> For me the following steps work:
> 
> 1. Verify that Terminal has Full Disk Access: System Preference > Security & Privacy > Privacy > Full Disk Access
> 2. Run `sudo kextunload -b io.macfuse.filesystems.macfuse`
> 
> macOS 12.1

Hello bro. I tried this and it works only for one time, next time when I want to open volume again from USB it resets. Veracrypt has full disk access and I have to run command everytime I want to open volume. Any other way to do it properly and once and for all? Im a bit tired running this command :(

MacOS 13.2.1

**Comment by 4pastor** [comment]
[GuillaumeHullin](https://github.com/GuillaumeHullin), I have tried that link and got the result below:

Last login: Sat Apr 15 13:54:02 on console
pastortheconstantgardener@Calvins-Air ~ % sudo kextunload -b io.macfuse.filesystems.macfuse
Password:
Executing: /usr/bin/kmutil unload -b io.macfuse.filesystems.macfuse
Error Domain=KMErrorDomain Code=71 "Kernel request failed: (libkern/kext) not found (-603947002)" UserInfo={NSLocalizedDescription=Kernel request failed: (libkern/kext) not found (-603947002)}
pastortheconstantgardener@Calvins-Air ~ %

**Comment by martinpacesa** [comment]
I have the same issue as above

**Comment by bfleischer** [comment]
macFUSE 4.4.3 contains a workaround to deal with the

    mount_macfuse: the macFUSE kernel extension is not loaded 

 error (even though the kernel extension is loaded). 

@4pastor It seems the kernel extension is not loaded on your Mac. That would explain why it cannot be unloaded. Could you check with `kextstat` whether the extension is actually loaded before running the unload command?

**Comment by 4pastor** [comment]
Thank you, but this case is resolved. I found what I needed to do and it's loading now.

Thanks for your kind support!

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by SofijaErkin** [comment]
The [bro's answer](https://github.com/osxfuse/osxfuse/issues/851#issuecomment-1052261655) fixed my same problem, please refer. @GuillaumeHullin thanks a lot !

**Comment by SofijaErkin** [comment]
> The [bro's answer](https://github.com/osxfuse/osxfuse/issues/851#issuecomment-1052261655) fixed my same problem, please refer. @GuillaumeHullin thanks a lot !

This problem occurs again after I fixed that. Again, this SOP again fixed that trouble.
